LinuxFunctionsStacks.json

```diff
@@ -1,5 +1,39 @@
 {
     "value": [
+        {
+            "id": null,
+            "name": "dotnet-isolated",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=LinuxFunctions",
+            "properties": {
+                "name": "dotnet-isolated",
+                "display": ".NET Core Isolated",
```

Comment:
Please update these fields to match https://github.com/Azure/azure-functions-ux/blob/dev/server/src/stacks/2020-10-01/stacks/function-app-stacks/Dotnet.ts
Thanks!